projects/plugins/jetpack-mu-wpcom/src/features/coming-soon/fallback-coming-soon-page.php

```diff
@@ -0,0 +1,295 @@
+<?php
+/**
+ * Coming Soon fallback page.
+ *
+ * This page is used when the site is set to Coming Soon mode, but no
+ * Coming Soon page id has been set.
+ *
+ * @package A8C\FSE\Coming_soon
+ */
+
+namespace A8C\FSE\Coming_soon;
+
+/**
+ * Returns the current locale
+ *
+ * @return string Language slug
+ */
+function get_current_locale() {
+	$language = function_exists( 'get_blog_lang_code' ) ? get_blog_lang_code() : get_locale();
+	return \A8C\FSE\Common\get_iso_639_locale( $language );
```

Comment:
Should this function be copied in from ETK too, so we don't depend on ETK being installed?

Reply:
Hmm this might be a bit tricky once we start migrating more things. For straightforward stuff like this how does b721cdbd9fcaffb96ebe558bcbb0ccb3cd0c3f39 look for now?

Reply:
Seems good to me. Is there a test we could copy over along with it? Or should we write one?

Reply:
There wasn't a test to move over, added one in e7281aa63cf33165a534a694b60d6f73a9bfb1a3